docs/configuration.md

```diff
@@ -0,0 +1,156 @@
+
+`provenance_url_pattern`: This provides a URL pattern to link to the source of a package. The `{feedstock}` placeholder is replaced with the feedstock name.
+```toml
+provenance_url_pattern = "https://github.com/conda-forge/{feedstock}-feedstock"
+```
+---
+
+`package_filter`: This property provides mechanism to filter packages by name or prefix.
+```toml
+# allows packages with names "r", "rpy2", "rstudio" and prefixes "r-", "_r-", "mro-"
+package_filter = { allowed_names = ["r", "rpy2", "rstudio"], allowed_prefixes = ["r-", "_r-", "mro-"] }
+```
+---
+
+`supports_broken_label`: This flag indicates that the channel supports a label called "broken" for releases that are not working (yanked).
+```toml
+supports_broken_label = true
+```
+---
+
+`dashboards`: List of dashboards that should be linked from the package browser. Dashboards are defined in the global `dashboards` section (see below).
+```toml
+dashboards = ["anaconda", "ghcr"]
+```
+---
+
+`auth_profile` The name of the authentication profile to use for this channel.
+TODO: remove this because the auth config is revised.
+```toml
+auth_profile = "my_auth_profile"
+```
+---
+
+`metadata_retrieval`: This key indicates how metadata for a package will be retrieved. It supports two options:
+- `streamed`: Currently only supports .conda packages.
+- `oci_with_streamed_fallback`: Tries to use the OCI registry first, then falls back to streamed metadata.
+```toml
+metadata_retrieval = "streamed"
+```
+---
+
+`override_extensions`: This is used to define extensions of conda package that override auto detection.
+Possible choices are `[".conda", ".tar.bz2"]`.
+```toml
+override_extensions = [".conda"]
+```
+
+## Configuring Dashboards
+
+Dashboards are related applications that are linked from the package browser. They are defined in the `[dashboards]` section with a unique name and a `url_pattern`.
+
+```toml
+[dashboards]
+anaconda = { url_pattern = "https://anaconda.org/{channel}/{name}/files?version={version}" }
+ghcr = { url_pattern = "https://github.com/orgs/channel-mirrors/packages/container/package/{channel}%2F{subdir}%2F{name}" }
+prefix = { url_pattern = "https://prefix.dev/channels/{channel}/packages/{name}" }
+```
+The `url_pattern` can contain placeholders that are encompassed in curly braces `{}`. These placeholders get replaced with the relevant value when generating the URLs. Current available placeholders include `{channel}`, `{name}`, `{version}`, and `{subdir}`. 
+
+Note that the `app_config.toml` file is loaded by the application at runtime. Therefore, any changes made while the app is running will take effect only after you restart the application. Ensure to follow the correct syntax to avoid runtime errors.
+
+## Complete Schema
+Refer to [app_config.py](../app_config.py) for the complete configuration schema.
+
+You can configure your IDE to use the [app_config.schema.json](../app_config.schema.json) file for auto-completion and validation of the `app_config.toml` file.
```

Comment:
How about adding a comment to your toml like this 

```
#:schema https://url.to/schema.json
```

See https://github.com/prefix-dev/pixi/blob/42c3a782ab7ece857c89096aa018c36db3f3c5a5/schema/examples/valid/full.toml#L1

At least for vscode it will automatically detect it then (given the even better toml extension is installed)

Reply:
done 05aec90c4c33887a555c388435200ac07310cf97